src/Helpers/UtilsGeneralHelper.php

```diff
@@ -41,98 +41,69 @@ public static function getProjectVersion(): string
 	/**
 	 * Method that returns listing page url.
 	 *
-	 * @return string
-	 */
-	public static function getListingPageUrl(): string
-	{
-		$page = UtilsConfig::SLUG_ADMIN;
-
-		return \get_admin_url(null, "admin.php?page={$page}");
-	}
-
-	/**
-	 * Method that returns form settings page url.
-	 *
-	 * @param string $formId Form ID.
 	 * @param string $type Type key.
+	 * @param string $formId Form ID.
+	 * @param string $parent Parent key.
+	 * @param string $page Top page key.
 	 *
 	 * @return string
 	 */
-	public static function getSettingsPageUrl(string $formId, string $type): string
+	public static function getListingPageUrl(string $type = '', string $formId = '', string $parent = '', string $page = UtilsConfig::SLUG_ADMIN): string
 	{
-		$page = UtilsConfig::SLUG_ADMIN_SETTINGS;
 		$typeKey = '';
+		$formIdKey = '';
+		$parentKey = '';
 
 		if (!empty($type)) {
 			$typeKey = "&type={$type}";
 		}
 
-		return \get_admin_url(null, "admin.php?page={$page}&formId={$formId}{$typeKey}");
-	}
-
-	/**
-	 * Method that returns form settings global page url.
-	 *
-	 * @param string $type Type key.
-	 *
-	 * @return string
-	 */
-	public static function getSettingsGlobalPageUrl(string $type): string
-	{
-		$page = UtilsConfig::SLUG_ADMIN_SETTINGS_GLOBAL;
-		$typeKey = '';
+		if (!empty($formId)) {
+			$formIdKey = "&formId={$formId}";
+		}
 
-		if (!empty($type)) {
-			$typeKey = "&type={$type}";
+		if (!empty($parent)) {
+			$parentKey = "&parent={$parent}";
 		}
 
-		return \get_admin_url(null, "admin.php?page={$page}{$typeKey}");
+		return \get_admin_url(null, "admin.php?page={$page}{$typeKey}{$formIdKey}{$parentKey}");
```

Comment:
Why not use something like:
```suggestion
		$args = [];
		if (!empty($parent)) {
			$args[] = ['parent' => $parent];
		}
		return \add_query_arg($args, get_admin_url(null, 'admin.php'));
```

Reply:
or just
```php
$args['parent'] = $parent;
```
😄